node/src/components/consensus/era_supervisor.rs

```diff
@@ -430,7 +430,9 @@ where
             &self.era_supervisor.public_signing_key,
             self.rng,
         );
-        let mut effects = responder.respond(signature).ignore();
+        let mut effects = responder
+            .respond((self.era_supervisor.public_signing_key, signature))
```

Comment:
Would it be cleaner to respond with a `FinalitySignature` here, rather than a tuple?

Reply:
Yeah, why not. It's a bit of redundant information but sure.